src/client/Header.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+
+const Header = () => {
+	return(
+		<div className="ui three item menu borderless">
```

Comment:
To match with every other Javascript file we have can this use spaces instead of tabs please